src/proto/mod.rs

```diff
@@ -11,6 +11,7 @@ pub mod response;
 
 pub use self::caps::{Capability, NegotiationVersion};
 pub use self::chan::ChannelExt;
+pub use self::colors::FormattedString;
```

Comment:
You'll need to add `pub mod colors` above for the Rust compiler to compile the module you wrote. This is why the Travis build failed.